cmd/mtls.go

```diff
@@ -101,5 +101,6 @@ func init() {
 	MTLSCmd.MarkFlagRequired("kubernetes")
 	MTLSCmd.AddCommand(ExportCMD)
 	MTLSCmd.AddCommand(ExpiryCMD)
+	MTLSCmd.AddCommand(RenewCertificateCmd)
```

Comment:
nit
```suggestion
	MTLSCmd.AddCommand(RenewCertificateCMD)
```
https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Since we are already using mixed conventions, we can stick to the recommended one.